src/Domain/Fixture/SubextensionManager.php

```diff
@@ -95,7 +95,7 @@ public function __construct(ConfigLoader $config_loader, Filesystem $filesystem,
    *   The package manager.
    */
   public function initializeTopLevelExtensions(PackageManager $package_manager): void {
-    foreach ($package_manager->getAll() as $package_name => $package) {
+    foreach ($package_manager->getCompanyPackages() as $package_name => $package) {
```

Comment:
Could there ever be a reason to add a Drupal module as a dependency? What if AMA used this to install Paragraphs, for example? In any case, based solely on the `packages.yml` schema and documentation, I would expect it to apply to dependencies, too, if I were a user.

```suggestion
    foreach ($package_manager-> getAll() as $package_name => $package) {
```

Reply:
Since we are already adding dependencies in the "fixDefaultDependencies" stage of fixture creation, would it make any difference? The dependency will be present in the fixture anyways.

Reply:
Oh, I was mixed up about what code I was looking at here. Yes, this is fine. 🙂️